TODO.md

```diff
@@ -0,0 +1,28 @@
+Systers Portal TODO 
+===================
+
+This file is that medium that is used to keep you updated on the exiciting features we are working on or what we are bringing to the Systers Portal next! 
+
+Website: http://portal.systers.org
+
+Project page: http://systers.github.io/portal/
+
+
+TODO LIST
+--------------
+
+### 1. ADD SUPPORT FOR ANACONDA PYTHON
+## Description
+Contributors/developers who use anaconda python, would be able to to work on Portal using the python distribution. The first release shall include anaconda python 3.6.
```

Comment:
Contributors who use Anaconda Python, would be able to to work on the Portal using this Python distribution. The first release will include Anaconda Python 3.6.